SUMMARY.md

```diff
@@ -73,6 +73,7 @@
   * [Mine](https://mine.treasure.lol)
   * [Discord](https://discord.gg/treasuredao)
   * [Twitter](https://twitter.com/Treasure\_NFT)
+  * [Github] (https://github.com/TreasureProject)
```

Comment:
Small nit - could you update to "GitHub" with the capital H? Thanks for contributing!

Reply:
Hi, 

I made the fix. Since it was minor change, amended it instead of making a new commit.